request.go

```
@@ -18,20 +18,20 @@ type HTTPClient interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
+var stdHTTPClientInterface HTTPClient = &http.Client{}
```

golangci-lint (1.64.8)

var `stdHTTPClientInterface` is unused

(unused)